testsuite/tests/ui/apiap/test_backend.py

```diff
@@ -55,7 +56,7 @@ def test_delete_backend(login, navigator, threescale, custom_backend):
     backend = custom_backend(autoclean=False)
     edit = navigator.navigate(BackendEditView, backend=backend)
     edit.delete()
-    backend = threescale.backends.read_by_name(backend.entity_name)
+    backend = resilient.resource_read_by_name(threescale.backends, backend.entity_name)
```

Comment:
Shouldn't the back off be reverted here? I thought we are testing that the backend no longer exist.

Reply:
@pehala should be reverted. In case of deletion test basic API call is used